frontend/src/components/History/HistoryTable.jsx

```diff
@@ -0,0 +1,335 @@
+import { Button, Form, OverlayTrigger, Table, Tooltip } from "react-bootstrap";
+
+import { DebouncedInput, GrayBox, H4 } from "../ui";
+import DataCard from "../DataCard/DataCard";
+import { useHistory, useVerifyLabel } from "../../hooks";
+import { PROJECT_USES_IRR } from "../../store";
+
+const defaultColumns = {
+    first: [
+        {
+            cell: ({ row }) => {
+                return row.getCanExpand() ? (
+                    <button
+                        className="unstyled-button p-1"
+                        style={{ fontSize: "0.5rem", width: "24px" }}
+                        {...{
+                            onClick: row.getToggleExpandedHandler()
+                        }}
+                    >
+                        {row.getIsExpanded() ? "▼" : "▶"}
+                    </button>
+                ) : null;
+            },
+            header: () => null,
+            id: "Expander"
+        },
+        {
+            accessorKey: "data",
+            filterFn: "includesString",
+            header: "Data",
+            id: "Data",
+            sortingFn: "alphanumeric"
+        },
+        {
+            accessorKey: "label",
+            filterFn: "includesString",
+            header: "Label",
+            id: "Current Label",
+            sortingFn: "alphanumeric"
+        },
+        {
+            accessorKey: "profile",
+            filterFn: "includesString",
+            header: "Labeled By",
+            id: "Labeled By",
+            sortingFn: "alphanumeric"
+        },
+        {
+            accessorKey: "timestamp",
+            filterFn: "includesString",
+            header: "Timestamp",
+            id: "Timestamp",
+            sortingFn: "datetime"
+        },
+    ],
+    last: [
+        {
+            accessorKey: "verified_by",
+            filterFn: "includesString",
+            header: "Verified By",
+            id: "Verified By",
+            sortingFn: "alphanumeric"
+        },
+        {
+            accessorKey: "pre_loaded",
+            filterFn: "includesString",
+            header: "Pre-Loaded",
+            id: "Pre-loaded",
+            sortingFn: "alphanumeric"
+        }
+    ]
+};
+
+const HistoryTable = () => {
+    const [columnResizeMode, setColumnResizeMode] = useState("onChange");
+    const [columnVisibility, setColumnVisibility] = useState({});
+    const [unlabeled, setUnlabeled] = useState(false);
+
+    const { data: historyData } = useHistory(unlabeled);
+    const { mutate: verifyLabel } = useVerifyLabel();
+
+    const metadataColumnsAccessorKeys = [];
+    if (historyData) {
+        historyData.data.forEach((data) => {
+            if (data.formattedMetadata)
+                Object.keys(data.formattedMetadata).forEach((metadataColumn) =>
+                    !metadataColumnsAccessorKeys.includes(metadataColumn) ? metadataColumnsAccessorKeys.push(metadataColumn) : null
+                );
+        });
+    }
+
+    const table = useReactTable({
```

Comment:
So I went to a project with many labels, clicked "unlabeled" and labeled an item. It worked fine. However, when I tried to label anything else, I got the following error (which kept occurring after a refresh of the page)

```
backend              | Internal Server Error: /api/modify_label/9394/
backend              | Traceback (most recent call last):
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/fields/__init__.py", line 1823, in get_prep_value
backend              |     return int(value)
backend              | ValueError: invalid literal for int() with base 10: ''
backend              | 
backend              | The above exception was the direct cause of the following exception:
backend              | 
backend              | Traceback (most recent call last):
backend              |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
backend              |     response = get_response(request)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
backend              |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
backend              |     return view_func(*args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 70, in view
backend              |     return self.dispatch(request, *args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
backend              |     response = self.handle_exception(exc)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
backend              |     self.raise_uncaught_exception(exc)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
backend              |     raise exc
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
backend              |     response = handler(request, *args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/decorators.py", line 50, in handler
backend              |     return func(*args, **kwargs)
backend              |   File "/code/core/views/api_annotate.py", line 441, in modify_label
backend              |     old_label = Label.objects.get(pk=request.data["oldLabelID"])
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
backend              |     return getattr(self.get_queryset(), name)(*args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 424, in get
backend              |     clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 941, in filter
backend              |     return self._filter_or_exclude(False, args, kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 961, in _filter_or_exclude
backend              |     clone._filter_or_exclude_inplace(negate, args, kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 968, in _filter_or_exclude_inplace
backend              |     self._query.add_q(Q(*args, **kwargs))
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1393, in add_q
backend              |     clause, _ = self._add_q(q_object, self.used_aliases)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1412, in _add_q
backend              |     child_clause, needed_inner = self.build_filter(
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1347, in build_filter
backend              |     condition = self.build_lookup(lookups, col, value)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1193, in build_lookup
backend              |     lookup = lookup_class(lhs, rhs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/lookups.py", line 25, in __init__
backend              |     self.rhs = self.get_prep_lookup()
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/lookups.py", line 77, in get_prep_lookup
backend              |     return self.lhs.output_field.get_prep_value(self.rhs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/db/models/fields/__init__.py", line 1825, in get_prep_value
backend              |     raise e.__class__(
backend              | ValueError: Field 'id' expected a number but got ''.
```